lib/ClangImporter/ClangImporter.cpp

```diff
@@ -1022,6 +1022,16 @@ bool ClangImporter::isModuleImported(const clang::Module *M) {
   return M->NameVisibility == clang::Module::NameVisibilityKind::AllVisible;
 }
 
+bool ClangImporter::canImportModule(std::pair<Identifier, SourceLoc> moduleID) {
+  // Look up the top-level module to see if it exists.
+  // FIXME: This only works with top-level modules.  To support submodules, this
+  // should be replaced by checking the result of `loadModule`.
```

Comment:
I'm not sure this is the correct planned fix, but since we're only allowing users to check top-level modules for now it seems like an acceptable limitation. Maybe just delete the "To support submodules" part?